ethportal-peertest/src/utils.rs

```diff
@@ -29,7 +29,7 @@ where
     Fut: Future<Output = Result<O>>,
 {
     // If content is absent an error will be returned.
-    for counter in 0..60 {
```

Comment:
I think this made some tests flaky, at least when running on circleci. The test that is frequently failing (with "Retried too many times" panic) is `peertest_history_offer_propagates_gossip_multiple_large_content_values`.

Test passes fine when I run it locally.

Reply:
Hmmm, thanks for the heads up. I'll look into it

Reply:
Well, I made this change since it's a real pain to wait for a minute if the test fails locally. But, I have no problem with you reverting the `10` -> `60` in #1515 . It's fairly simple to make this change when testing locally if it becomes a pain in the future